python/plugins/phidata/composio_phidata/toolset.py

```diff
@@ -28,39 +28,60 @@ def _wrap_tool(
         self,
         schema: t.Dict,
         entity_id: t.Optional[str] = None,
-    ) -> Function:
+    ) -> Toolkit:
         """
-        Wrap composio tool as Phidata `Function` object.
+        Wrap composio tool as Phidata `Toolkit` object.
         """
         name = schema["name"]
         description = schema["description"]
         parameters = schema["parameters"]
-
+        
+        # Create a new Toolkit instance
+        toolkit = Toolkit(name=name)
+        
+        @validate_call
         def function(**kwargs: t.Any) -> str:
-            """Composio tool wrapped as Phidata `Function`."""
+            """Composio tool wrapped as Phidata `Function`.
+            
+            Args:
+                **kwargs: Function parameters based on the schema
+            
+            Returns:
+                str: JSON string containing the function execution result
+            """
             return json.dumps(
                 self.execute_action(
                     action=Action(value=name),
                     params=kwargs,
                     entity_id=entity_id or self.entity_id,
                 )
             )
-
-        return Function(
-            name=name,
-            description=description,
-            parameters=parameters,
-            entrypoint=validate_call(function),
-        )
+        
+        # Set function docstring from schema
+        param_docs = []
+        if "properties" in parameters:
+            for param_name, param_info in parameters["properties"].items():
+                param_desc = param_info.get("description", "No description available")
+                param_type = param_info.get("type", "any")
+                param_docs.append(f":param {param_name}: {param_desc} ({param_type})")
```

Comment:
The docstring style mixes `:param` style with Google-style docstrings (Args/Returns sections). Consider standardizing on one format, preferably Google-style since it's more readable and matches the new sections you've added.